src/js/angular/core/services/graphql.service.js

```diff
@@ -42,9 +43,20 @@ function GraphqlService(GraphqlRestService) {
             .then((response) => endpointsInfoListMapper(response.data));
     };
 
+    const getGraphqlEndpointConfiguration = (repositoryId, endpoint) => {
+        return GraphqlRestService.getGraphqlEndpointConfiguration(repositoryId, endpoint)
+            .then((response) => new GraphqlEndpointConfiguration(response.data.configs));
```

Comment:
As already mentioned in another comment, here it's better to be returned only the settings object of GraphqlEndpointConfigurationSettings class

Reply:
fixed by using the new mappers and models